### PR TITLE
Fix: update Terraform ComponentDefinition development doc

### PR DESCRIPTION
**Pull request by zzxwill**
Update the docs of developing a Terraform typed ComponentDefinition,
and added the section of automatically generated official site docs.

Signed-off-by: Zheng Xi Zhou <zzxwill@gmail.com>

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/components/component-terraform.md [hunk 1]
```diff
@@ -114,3 +75,36 @@ $ vela show alibaba-eip
 
 如果表格能正常出来，ComponentDefinition 应该就可以工作了。更进一步，你可以通过文档[部署云资源](../../end-user/components/cloud-services/provider-and-consume-cloud-services)创建一个实际的 EIP 来验证。
 
+## 生成文档
+
+我们鼓励你为你的 ComponentDefinition 生成文档并提交给 [KubeVela官方网站]（https://github.com/oam-dev/kubevela.io）。
+
+```shell
+$ vela def gen-doc alibaba-eip -n vela-system
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/components/component-terraform.md]:
```suggestion
$ vela def doc-gen alibaba-eip -n vela-system
```

这个命令修正以后再合并。

File: docs/platform-engineers/components/component-terraform.md [hunk 2]
```diff
@@ -116,3 +73,37 @@ $ vela show alibaba-eip
 
 If the tables display, the ComponentDefinition should work. To take a step further, you can verify it by provision an actual EIP instance per
 the doc [Provision cloud resources](../../end-user/components/cloud-services/provider-and-consume-cloud-services#provision-cloud-resources).
+
+## Generate documentation
+
+You are encouraged to generate the documentation for your ComponentDefinition and submit it to [KubeVela official site](https://github.com/oam-dev/kubevela.io).
+
+```shell
+$ vela def gen-doc alibaba-eip -n vela-system
```
Comment (zzxwill) [inline docs/platform-engineers/components/component-terraform.md]:
```suggestion
$ vela def doc-gen alibaba-eip -n vela-system
```

File: i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/components/component-terraform.md (hunk 1 above)
Comment (zzxwill) [reply i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/components/component-terraform.md]: Fixing it in https://github.com/oam-dev/kubevela/pull/3078

**Review by wonderflow — APPROVED** [review]

**Comment by wonderflow** [comment]
https://github.com/oam-dev/kubevela/pull/3078 合并了以后再merge

**Pull request merged**